src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilterArgumentCollection.java

```diff
@@ -0,0 +1,100 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+import org.broadinstitute.barclay.argparser.Argument;
+
+import java.io.Serializable;
+
+public final class PSFilterArgumentCollection implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Argument(doc = "Set if the input BAM is aligned to the host",
+            fullName = "isHostAligned",
+            optional = true)
+    public boolean alignedInput = false;
+    @Argument(doc = "Path to kmer library generated with PathSeqBuildKmers. Skipped if not specified.",
+            fullName = "kmerLibraryPath",
+            optional = true)
+    public String kmerLibPath = null;
+    @Argument(doc = "Skip read quality and low-complexity filtering",
+            fullName = "skipFilters",
+            optional = true)
+    public boolean skipFilters = false;
+    @Argument(doc = "Keep only clipped reads with length at least equal to the specified value",
+            fullName = "minClippedReadLength",
+            minValue = 0,
+            minRecommendedValue = 31,
+            optional = true)
+    public int minReadLength = 31;
+    @Argument(doc = "Max allowable fraction of ambiguous bases",
+            fullName = "maxAmbiguousBaseFraction",
+            minValue = 0.0,
+            maxValue = 1.0,
+            optional = true)
+    public double fracNThreshold = 0.03;
+    @Argument(doc = "Bases below this read quality will be transformed into 'N'",
+            fullName = "minBaseQuality",
+            minValue = 1,
+            optional = true)
+    public int qualPhredThresh = 15;
+    @Argument(doc = "Quality score trimmer threshold",
+            fullName = "readTrimmerThreshold",
+            minValue = 1,
+            optional = true)
+    public int readTrimThresh = 15;
+    @Argument(doc = "Base quality to assign DUST masked bases",
+            fullName = "dustPhred",
+            optional = true)
+    public int dustMask = 2;
+    @Argument(doc = "DUST window size",
+            fullName = "dustWindowSize",
+            optional = true)
+    public int dustW = 64;
+    @Argument(doc = "DUST score threshold",
+            fullName = "dustTScore",
+            optional = true)
+    public double dustT = 20.0;
+    @Argument(doc = "Host alignment coverage score threshold, in bp",
+            fullName = "minCoverage",
+            minValue = 1,
+            optional = true)
+    public int minCoverage = 31;
+    @Argument(doc = "Host alignment identity score threshold, in bp",
+            fullName = "minIdentity",
+            minValue = 1,
+            optional = true)
+    public int minIdentity = 30;
+    @Argument(doc = "Host kmer count threshold.",
+            fullName = "hostKmerThreshold",
+            minValue = 1,
+            optional = true)
+    public int hostKmerThresh = 1;
+    @Argument(doc = "The bwa mem index image file of the host reference, distributed on each executor.",
```

Comment:
This seems to be a point of confusion with users trying to run the other tools that require BWA indices, so I'd be very explicit here that this needs to be a path on the local filesystem of each worker node in the cluster.